web/modules/custom/dpl_related_content/dpl_related_content.module

```diff
@@ -144,6 +154,32 @@ function _dpl_related_content_filter_paragraph(array $variables, string $list_st
   return $variables;
 }
 
+/**
+ * Used as dynamic list options (allowed_values_function) for a field.
+ *
+ * See field.storage.paragraph.field_filter_content_types.yml for more info.
+ *
+ * @return array<int|string, \Drupal\Core\StringTranslation\TranslatableMarkup|string|null>
+ *   The options, used in field_filter_content_types dropdown.
+ */
+function dpl_related_content_field_filter_content_types_options(): array {
+  $content_types = ['event' => t('Event', [], ['context' => 'DPL admin UX'])];
```

Comment:
```suggestion
  $content_types = ['event' => t('Event', [], ['context' => 'DPL Related content'])];
```

We normally use the module name as the context.

Reply:
I kinda disagree - this is part of the admin interface, hence why I used that context